ui/perf.html

```diff
@@ -72,14 +72,18 @@
   <script src="js/models/job.js"></script>
   <script src="js/models/runnable_job.js"></script>
   <script src="js/models/resultset.js"></script>
+  <script src="js/models/user.js"></script>
   <script src="js/perf.js"></script>
   <script src="js/controllers/perf/compare.js"></script>
   <script src="js/controllers/perf/graphs.js"></script>
   <script src="js/controllers/perf/alerts.js"></script>
+  <script src="js/directives/treeherder/persona.js"></script>
   <script src="js/directives/perf/compare.js"></script>
   <script src="js/perfapp.js"></script>
   <script src="js/filters.js"></script>
   <!-- endbuild -->
 
+  <script src="https://login.persona.org/include.js"></script>
```

Comment:
Even if the JS snippet was hosted on treeherder.m.o, the Persona IDP is still required for authentication, so people would still not be able to log in.